buildpipeline/pipeline.json

```diff
@@ -545,7 +545,7 @@
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
             "PB_DockerTag": "opensuse421_prereqs_v3",
-            "PB_TargetQueue": "suse.421.amd64"
```

Comment:
We should just coordinate the changes with coreclr and core-setup. @gkhanna79 do you guys still build for opensuse42.1?